applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -102,6 +102,7 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.FLAG_VARIABLE)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE)              # Distance function nodal values
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE_GRADIENT)     # Distance gradient nodal values
+        self.main_model_part.AddNodalSolutionStepVariable(KratosCFD.DIVERGENCE)
```

Comment:
I am quite sure you don’t need this

Also adding variables is very critical/sensitive bcs it increases the already high mem-consumption

Reply:
Ok, as I see now I had not fully understood the behavior of variables in Kratos. Thank you very much for the hints!